event-app-api/merge_intervals/merge_intervals.test.js

```diff
@@ -0,0 +1,200 @@
+import { mergeIntervals, findGaps } from "./merge_intervals.js";
+
+describe("Merge Intervals", () => {
+  test("Events with overlapping times returns one start and end time", () => {
+    const testEvents = [
+      {
+        starttime: 1679990400000,
+        endtime: 1680076800000,
```

Comment:
Can you put a comment next to all of these epoch times with the date in English? The numbers are hard to read easily

Reply:
Okay, I'll do that